src/brainbrowser/surface-viewer/modules/rendering.js

```diff
@@ -211,14 +217,16 @@ BrainBrowser.SurfaceViewer.modules.rendering = function(viewer) {
   * @doc function
   * @name viewer.rendering:setClearColor
   * @param {number} color A hexadecimal number representing the RGB color to use.
+  * @param {number} define the opacity.
   * @description
   * Updates the clear color of the viewer.
   * ```js
   * viewer.setClearColor(0xFF0000);
   * ```
   */
-  viewer.setClearColor = function(color)  {
-    renderer.setClearColor(color, 1.0);
+  viewer.setClearColor = function(color,alpha)  {
+    alpha = typeof(alpha) !== 'undefined' ? alpha : 1.0;
```

Comment:
I assume these changes relate to the transparency issue you mentioned?

Here, I think it might be clearer to just write:
```javascript
if (alpha === undefined) alpha = 1;
```
Wouldn't that do the same thing?